src/US_ZIPCode/Lookup.php

```diff
@@ -27,17 +29,23 @@ public function __construct($city = null, $state = null, $zipcode = null) {
         $this->city = $city;
         $this->state = $state;
         $this->zipcode = $zipcode;
+        $this->customParamArray = array();
+        $this->jsonArray = array();
     }
 
     #[\ReturnTypeWillChange]
     public function jsonSerialize()
     {
-        return array(
+        $this->jsonArray = array(
```

Comment:
Why is the transitory serialization array stored on the Lookup object (`$this->jsonArray`)? This is particularly odd because it seems it would only be fully valid after `jsonSerialize` is called, and potentially invalid thereafter if any setters are called.

Reply:
I had other plans for jsonArray earlier in the development process to make it a more permanent object, but it ended up being replaced in that aspect by customParamArray. You are correct, that does not need to be a member of the Lookup object. That should be resolved in my latest commit. Thanks for pointing that out.